src/test/java/mandel/fallingsand/SandTest.java

```diff
@@ -0,0 +1,80 @@
+package mandel.fallingsand;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SandTest
```

Comment:
Write a test that calls `sand.put()` for a whole row and then makes sure that every piece of sand moves down.